extensions-core/kafka-extraction-namespace/src/test/java/io/druid/query/lookup/KafkaLookupExtractorFactoryTest.java

```diff
@@ -118,7 +117,7 @@ public void testCacheKeyScramblesOnNewData()
         Assert.assertFalse(Arrays.equals(byteArray, myKey));
       }
       byteArrays.add(myKey);
-      events.incrementAndGet();
+      factory.onNewEntry("", "");
```

Comment:
This is done instead of directly manipulating the event count. Actually, completely splitting up the kafka-interacting part from the lookup manager-interacting one would be even better in terms of tests.